imports/node-app/index.js

```diff
@@ -15,28 +11,25 @@ if (!ROOT_URL) throw new Error("You must set ROOT_URL");
 const app = new ReactionNodeApp({
   debug: true,
   context: {
-    mutations,
-    queries,
-    rootUrl: ROOT_URL
+    rootUrl: ROOT_URL,
```

Comment:
These were likely intentionally sorted in alphabetical order.

Reply:
Good catch. I removed them, then later added back in the wrong place.